src/main.py

```diff
@@ -1,17 +1,28 @@
-from flask import Flask, request
-
+from flask import Flask, request, Response , json 
+from store.sparql import SparqlStore
 
 app = Flask(__name__)
+storage = SparqlStore()
+jsonld = "application/ld+json"
```

Comment:
I like the use of a constant (good thinking) but:

- python contants are uppercase, `JSONLD` please.
- given this constant doesn't naturally sit in either file that's using it can you make a `constants.py` at the same level as `main.py` and import it from there please. Nice and obvious what it is then and encourages future folks to follow the same pattern with other mimetypes.